charts/ziti-controller/templates/configmap.yaml

```diff
@@ -311,3 +311,5 @@ data:
             options: { }
     {{- end }}
 
+    {{- .Values.additionalConfigs | toYaml | nindent 4 }}
```

Comment:
We'll have to do this separately for each top-level section we want to allow additional configs. It's because ziti will panic if, for example, Helm inputs include `additionalConfigs.acme` and the configmap template already inserted a top-level `acme`, and the config loader expects only one such property.

Reply:
Changed this to append additional configs at the end of the top level config blocks. This allows for configs within the blocks to be overridden, and also protects from a process panic.